transport/api/src/lib.rs

```diff
@@ -310,6 +310,7 @@ where
                                 }
                             }
                             PeerDiscovery::Ban(peer_id) => {
+                                debug!(peer = %peer_id, "Processing peer discovery event: Ban");
```

Comment:
This does not need to be here, since the termination processor is the `discovery.rs` which logs this at this level.